src/main/kotlin/no/nav/familie/ef/iverksett/arena/VedtakhendelseProducer.kt

```diff
@@ -0,0 +1,23 @@
+package no.nav.familie.ef.iverksett.arena
+
+import org.slf4j.LoggerFactory
+import org.springframework.jms.core.JmsTemplate
+import org.springframework.stereotype.Component
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Component
+class VedtakhendelseProducer(val jmsTemplate: JmsTemplate) {
+
+
+    private val logger = LoggerFactory.getLogger(javaClass)
+
+    @Transactional
+    fun produce(message: String) {
```

Comment:
Burde denne ta in ett objekt eller noe som vi serialiserer til json/riktig format?

Reply:
Tror det er lurt med et objekt her når vi faktisk skal implementere innholdet i overføringen.

Reply:
Ja, det er meningen at den skal ta inn objekt, men som Charlie sier, så tenker jeg det er lurt å ha den type endringer i egen PR.